packages/circuit-ui/components/NotificationBanner/NotificationBanner.tsx

```diff
@@ -172,12 +179,13 @@ const ResponsiveButton = styled(Button)(buttonStyles);
 const imageStyles = ({
   theme,
   image,
-}: { image: ImageProps } & StyleProps) => css`
+}: { image: NotificationImageProps } & StyleProps) => css`
   border-radius: 0 ${theme.borderRadius.mega} ${theme.borderRadius.mega} 0;
   min-width: 0;
   width: ${image.width || '200'}px;
   height: auto;
   object-fit: contain;
+  object-position: ${image.align || 'center'};
```

Comment:
https://caniuse.com/mdn-css_properties_object-position

But 🤷, this is a progressive enhancement

Reply:
We already use `object-fit` above which has similar browser support. 

> But 🤷, this is a progressive enhancement

Exactly :)